index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Recipe App</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--logo -->
+    <div class="logo">
+      <img
+        src="asset/Screenshot_2023-12-09_at_17-11-02_Deep_Dish_Pizza_Logo-removebg-preview.png"
+        alt="deep logo"
+      />
+    </div>
+    <!--Category-->
+    <section class="category-container">
+      <div class="category-container row">
+        <h2>Categories</h2>
+        <a href="#" class="category-btn">view all catergories</a>
+      </div>
+      <div class="category-cards column">
+        <a href="#" class="card-card">
+          <img src="" alt="" />
+          <h3>Breakfast</h3>
+        </a>
+        <a href="#" class="card-card">
+          <img src="" alt="" />
+          <h3>Snacks</h3> </a
+        ><a href="#" class="card-card">
+          <img src="" alt="" />
+          <h3>Drinks</h3>
+        </a>
+        <a href="#" class="card-card">
+          <img src="" alt="" />
+          <h3>Indian Cuisine</h3>
+        </a>
+      </div>
+    </section>
```

Comment:
All this is not necessary yet. The first part of the exercise is about showing one recipe and being able to add one recipe :)